ganga/GangaCore/Lib/Condor/Condor.py

```diff
@@ -797,4 +797,38 @@ def timedetails(self):
     updateMonitoringInformation = \
         staticmethod(updateMonitoringInformation)
 
+    def peek(self, filename=None, command=None):
+        job = self.getJobObject()
+
+        if not self.id:
+            logger.warning("Job %s not running" % job.id)
+            return False
+
+        idElementList = job.backend.id.split("#")
+        if 3 == len(idElementList):
+            if idElementList[1].find(".") != -1:
+                peekCommand = "condor_tail -f -name %s %s" % \
+                              (idElementList[0], idElementList[1])
```

Comment:
I suggest to use `f` strings instead as they are more readable. Same change in several other places in code.

```suggestion
                peekCommand = f"condor_tail -f -name {idElementList[0]} {idElementList[1]}"
```

Reply:
Done